src/utils/coingecko_types.ts

```diff
@@ -23,7 +23,10 @@ type CoinGeckoRates<F extends CoinGeckoVSCurrency> = {
  *   'https://api.coingecko.com/api/v3/simple/supported_vs_currencies' \
  *   -H 'accept: application/json'
  */
-const coinGeckoSupportedVSCurrencies = [
+export const coinGeckoSupportedVSCurrencies = [
```

Comment:
I think maybe some of the stuff in this file was supposed to go away... See "currencyIDs" in fiat_oracle_types instead.

Reply:
@matibat please confirm that this file is supposed to go away entirely. Thanks.

Reply:
That file has been removed in the latest commits